scripts/pylint_extensions_test.py

```diff
@@ -1973,3 +1974,121 @@ def test_well_formed_multi_line_docstring(self):
 
         with self.checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class SpaceBelowFileOverviewCheckerTests(unittest.TestCase):
```

Comment:
Also add a test where there's a unicode character.

Reply:
done

Reply:
I've also added some tests with cases having appropriate space between file overview and imports